### PR TITLE
Mix PAM - XEP-0405

### PR DESCRIPTION
**Pull request by stevekille**
Make Archiving Optional
Bump Namespace

### CODE REVIEW

File: xep-0405.xml [hunk 1]
```diff
@@ -199,7 +208,8 @@ This approach  enables flexible support of multiple clients for a MIX channel pa
     to='hag66@shakespeare.example/UUID-c8y/1573'
     type='result'>
   <query xmlns='http://jabber.org/protocol/disco#info'>
-    <feature var='urn:xmpp:mix:pam:0'/>
+    <feature var='urn:xmpp:mix:pam:1'/>
+     <feature var='urn:xmpp:mix:pam:1:archive'/>
```
Comment (lnjX) [inline xep-0405.xml]: [trivial] Here's one space too much

Comment (stevekille) [reply xep-0405.xml]: pedant!

Comment (lnjX) [reply xep-0405.xml]: Sorry, but it really doesn't look too good when the indentation is wrong.

Comment (stevekille) [reply xep-0405.xml]: I know!  I fixed it.  Thanks for pointing it out

**Pull request merged**